Automattic/wp-calypso
# Theme Showcase - Add trending tab to themes showcase

#### Changes proposed in this Pull Request

* This PR adds a "Trending" tab to the Themes Showcase. It follows the existing pattern for fetching recommended themes, but instead calls the v1.1 Themes endpoint, which supports a "sort" parameter which defaults to "trending". It also provides a small amount of general refactoring to future-proof this process for adding further tabs in the future.

#### Testing instructions



* Navigate to `/themes/{SITE_URL}` while logged in
* Verify that a "Trending" tab appears on the search tab bar.
* Verify that clicking on it presents a customized list of themes, different from that under "All" or "Recommended".



Related to #54088
